crates/astria-composer/src/metrics_init.rs

```diff
@@ -0,0 +1,234 @@
+//! Crate-specific metrics functionality.
+//!
+//! Registers metrics & lists constants to be used as metric names throughout crate.
+
+use metrics::{
+    describe_counter,
+    describe_gauge,
+    describe_histogram,
+    Unit,
+};
+
+/// Labels
+pub(crate) const ROLLUP_ID_LABEL: &str = "rollup_id";
```

Comment:
I feel like this can be removed for the time being.

Reply:
i think we should keep it for metrics to segregate the number of sequence actions submitted per rollup.